packages/web/src/scss/helpers/index.html

```diff
@@ -3,11 +3,28 @@
   <section class="docs-Section">
 
     <h2 class="docs-Heading">Accessibility</h2>
-    <p>
-      The following element is visually hidden, but still accessible by AT (use dev tools to check it's really there!).
-    </p>
 
-    <p class="accessibility-hidden">This text is visually hidden</p>
+    <div class="mb-400">
+      <p>
+        The following element is visually hidden, but still accessible by AT (use dev tools to check it's really there!).
+      </p>
+
+      <p class="accessibility-hidden">This text is visually hidden</p>
+    </div>
+
+    <div class="mb-400">
+      <p>Toggle display of element inside toggle depending on `aria-expanded`</p>
```

Comment:
```suggestion
      <p>Toggle display of an element inside the toggle button depending on `aria-expanded`</p>
```

(That's what my poor intuition says, I didn't consult Grammarly.)

Reply:
fixed